roles/engineblock/tasks/main.yml

```diff
@@ -64,6 +64,30 @@
   when:
     customlogo.stat.exists
 
+- name: Check if we have a custom background back image for the feedback page
+  local_action: stat path="{{ inventory_dir }}/files/background-back.svg"
+  register: customfeedbackbackground
+  become: false
+
+- name: Install environment specific background back image
+  copy:
+    src: "{{ inventory_dir }}/files/background-back.svg"
```

Comment:
could you put in in the subdirectory {{ inventory_dir }}/files/eb ? Makes it easier to differentiate between files for different applications

Reply:
:+1: